test/sys/mod.rs

```diff
@@ -26,6 +26,5 @@ mod test_uio;
 #[cfg(target_os = "linux")]
 mod test_epoll;
 mod test_pthread;
-#[cfg(any(target_os = "android",
-          target_os = "linux"))]
+#[cfg(not(target_os = "ios"))]
```

Comment:
This `#[cfg()]` statement is too broad.  Nix runs on many platforms.  You should opt-in, not opt-out.